### ISSUE TITLE
Modal dialogs should be `aria-modal="true"`

### ISSUE THREAD

**Issue by Malvoz** [thepassle/generic-components#53]
When a dialog traps focus (modal dialog) it should be [`aria-modal="true"`](https://www.w3.org/TR/wai-aria-1.2/#aria-modal).

**Comment by thepassle** [comment]
Fixed, thanks for reporting!
